### DISCUSSION TITLE
Mutliple go.mod's makes VSCode sad....

### DISCUSSION THREAD

**Discussion by jeff-mccoy** [zarf-dev/zarf#137] [Q&A]
Anyone else seen VSCode get all sad with multiple golang modules?  It seems multi-workspace support isn't really great so I can either: 

1. Temporarily rename the test/e2e/go.mod -> go.mods
2. Open only the subfolder cli 

Am I the only one getting this issue?

**Comment by RothAndrew** [comment]
I haven't seen any issues, though I almost exclusively use Goland. Happy to entertain alternatives if the current setup is causing issues.

**Comment by YrrepNoj** [comment]
I've been having the same woes with VSCode and have been opening directly to the cli subfolder. It makes me a little sad when I want to then edit the e2e tests and am too lazy to re-open the parent folder so I just end up using vim.

**Comment by RothAndrew** [comment]
I just noticed https://github.com/google/go-containerregistry has the `go.mod` in the root, and different golang codebases (like `crane`, and `gcrane` are in different subfolders.

We could do that.

**Comment by RothAndrew** [comment]
Created #146 to look at this.
